### PR TITLE
Updated Czech translation (customary)

### PR DESCRIPTION
**Pull request by Validbit**

### CODE REVIEW

File: res/values-cs/strings.xml [hunk 1]
```diff
@@ -43,11 +43,11 @@
   <string name="pref_autocapitalisation_title">Automatická kapitalizace</string>
   <string name="pref_autocapitalisation_summary">Stiskne Shift na začátku věty</string>
   <string name="pref_switch_input_immediate_title">Přepnout na posledně užívanou klávesnici</string>
-  <string name="pref_switch_input_immediate_summary">Jak bude klávesa pro přepnutí klávesnice reagovat</string>
+  <string name="pref_switch_input_immediate_summary">Namísto okna výběru klávesnice (u klávesy pro změnu klávesnice)</string>
```
Comment (Julow) [inline res/values-cs/strings.xml:46]: This might be too long on some devices.

**Review by Julow — APPROVED** [review]
Many thanks!

**Comment by Julow** [comment]
I've pushed a conflicting change, it's only removing strings: 1eea9e2

**Comment by Julow** [comment]
I've added more strings in ddc4ff1.

File: res/values-cs/strings.xml (hunk 1 above)
Comment (Validbit) [reply res/values-cs/strings.xml:46]: I rephrased it so it's even clearer what it does.

Should i revert this line and create new PR?

Comment (Julow) [reply res/values-cs/strings.xml:46]: You can modify this branch. You also need to pull the master branch. Happy to do it if the conflicts I created are too annoying to fix.

Comment (Validbit) [reply res/values-cs/strings.xml:46]: I see there were more changes after the translation so I manually updated the file and I'm enclosing it here
[values-cs.zip](https://github.com/Julow/Unexpected-Keyboard/files/12283274/values-cs.zip)

Comment (Julow) [reply res/values-cs/strings.xml]: I've pushed directly to your branch, thanks a lot and sorry for the conflicts.

Comment (Validbit) [reply res/values-cs/strings.xml]: no problem, thank you for the progress :)

**Pull request merged**